modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -13,7 +15,12 @@
 		premium += skyrat_premium
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
-
+	if(skyrat_amount_override)
+		for (var/item in products)
+			products[item] = 5
```

Comment:
This currently doesn't actually use the 'amount override' variable, it just forces 5 no matter what. If you put 'products[item] = skyrat_amount_override' then it'll actually use whatever we supply the vendor

Regardless, this is a bit of a troublesome way of doing it, as it sets ALL items to five - no more, no less.
I can look into helping improve it sometime later but git on phones SUCKS OH GOD

Reply:
Yea, I did think about checking if the item is a subtype of clothing and make sure there no armour or special tags for it before setting to 5-8 on every single vendor- though I've no idea how well that'd work.

Also, I can't believe I missed that! Will fix!